addon/components/paper-select/template.hbs

```diff
@@ -32,4 +32,21 @@
     {{yield opt term}}
   </PowerSelect>
 
-</md-input-container>
\ No newline at end of file
+  {{#unless this.hideAllMessages}}
+    <div class="md-errors-spacer" id={{concat this.inputElementId "-char-count"}}>
+      {{#if @maxlength}}
+        <div class="md-char-counter">{{this.renderCharCount}}</div>
+      {{/if}}
+    </div>
```

Comment:
I just realised that we can remove this whole char count div.
PaperSelect is not a control you can type characters in and `renderCharCount` is not even defined.